modules/userman/operations.go

```diff
@@ -0,0 +1,361 @@
+package userman
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"golang.org/x/crypto/bcrypt"
+
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/spaceuptech/space-cloud/model"
+	"github.com/spaceuptech/space-cloud/utils"
+)
+
+func (m *Module) Profile(ctx context.Context, token, dbType, project, id string) map[string]interface{} {
```

Comment:
Lets not return a map. Its makes the implementation unclear. Lets return status, result, err? Let result be a map.

Reply:
But then if there is no error, how do I return nil if I keep error as a string?

Reply:
Use type `error`